### PR TITLE
Improve browse layout for long file names

### PR DESCRIPTION
**Pull request by incon**
PR for issue #439

### CODE REVIEW

File: caddy/setup/browse.go [hunk 1]
```diff
@@ -123,19 +123,34 @@ h1 {
 }
 
 table {
+	table-layout:fixed;
```
Comment (JasonSwindle) [inline caddy/setup/browse.go]: Should this not be:

```
table-layout: fixed;
width: 100%;
```
to match the style of  the others?

**Comment by incon** [comment]
More changes based on the existing layout

**Comment by mholt** [comment]
Hmmm, now it's not as responsive anymore. I kind of liked it before the extra changes.

I can take a stab at a better browse layout soon, just gotta get this patch done first.

**Comment by incon** [comment]
Mobile
screen shot 2015-12-30 at 07 25 15
Desktop
screen shot 2015-12-30 at 07 25 26

**Comment by mholt** [comment]
Nice, I'm liking the details though that make it better in the header and table cells - and the IE11 fix. Thanks. I'm almost finished with a revamped layout that makes better use of space, feels lighter, doesn't rely on unicode (good idea in gitter to use svg) and incorporates your subtle changes as well for better viewing experience.

But I'm having an issue with the SVGs, if you want to take a look: https://gist.github.com/mholt/3f6a441646383dede68f - something wrong with the fills I think.

**Pull request closed without merging**